src/main/java/seedu/address/storage/JsonAdaptedTag.java

```diff
@@ -1,48 +1,69 @@
 package seedu.address.storage;
```

Comment:
JsonAdaptedTag was used to store tags for person. For now shouldn't it be deprecated since we do not have that functionality?

Reply:
It's okay I think just keep it first..